public/app/features/panel/panellinks/link_srv.ts

```diff
@@ -71,45 +94,89 @@ export class LinkSrv {
     return info;
   }
 
-  getPanelLinkAnchorInfo(link, scopedVars) {
-    const info: any = {};
-    info.target = link.targetBlank ? '_blank' : '';
-    if (link.type === 'absolute') {
-      info.target = link.targetBlank ? '_blank' : '_self';
-      info.href = this.templateSrv.replace(link.url || '', scopedVars);
-      info.title = this.templateSrv.replace(link.title || '', scopedVars);
-    } else if (link.url) {
-      info.href = link.url;
-      info.title = this.templateSrv.replace(link.title || '', scopedVars);
-    } else if (link.dashUri) {
-      info.href = 'dashboard/' + link.dashUri + '?';
-      info.title = this.templateSrv.replace(link.title || '', scopedVars);
-    } else {
-      info.title = this.templateSrv.replace(link.title || '', scopedVars);
-      const slug = kbn.slugifyForUrl(link.dashboard || '');
-      info.href = 'dashboard/db/' + slug + '?';
-    }
-
-    const params = {};
-
-    if (link.keepTime) {
-      const range = this.timeSrv.timeRangeForUrl();
-      params['from'] = range.from;
-      params['to'] = range.to;
-    }
+  getDataPointVars = (seriesName: string, valueTime: DateTime) => {
+    const timeRange = this.timeSrv.timeRange();
+    const targetAbsoluteTimeRange = transformAbsoluteTimeRange(timeRange, DataLinkDataPointRangeFactor);
+    const dataPointRangeQuery = toUrlParams({
+      from: dateTime(valueTime)
+        .subtract(targetAbsoluteTimeRange)
+        .valueOf(),
+      to: dateTime(valueTime)
+        .add(targetAbsoluteTimeRange)
+        .valueOf(),
+    });
```

Comment:
Think there is a missunderstanding here, the ${__value_time} should only interpolate to the time, not from & to. If the user wants that they will need to set &from=${__value_time}&to=${__value_time}

Reply:
Fixed